server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -134,11 +136,10 @@ public static Club getClub(DatabaseClient dbClient, String clubId) {
                                      .setName(row.getString(/* nameIndex= */ 1))
                                      .setClubId(clubId)
                                      .setDescription(row.getString(/* descriptionIndex= */ 2))
-                                     .setOwnerId(row.getString(/* ownerIdIndex= */ 3));                       
-      if (!row.isNull(/* contentWarningIndex =*/ 4)||
-          !row.getString(/* contentWarningIndex =*/ 4).isEmpty()) {
+                                     .setOwnerId(row.getString(/* ownerIdIndex= */ 3));
+      if (!row.isNull(/* contentWarningIndex =*/ 4)) {
         List<String> contentWarnings =
-            Arrays.asList(row.getString(/* contentWarningIndex =*/ 4).split("\\n"));
+            Arrays.asList(row.getString(/* contentWarningIndex =*/ 4).split("\\R"));
```

Comment:
The \\R is a little concerning here, since it's usually platform dependent. \r is a carriage return, while \n is a new line (which on most platforms includes a carriage return). If this works reliably we can call it good enough for now, but it might warrant a little bit of investigation (eg. maybe when we write the lines we should write \\R as well)

Reply:
Yeah you've got a point here. `\R` will match any linebreak character (https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) but we overtly set this linebreak character ourselves in club.save() (and it's set to `\n`). Because it is always the case that we set this delimiter to `/n`, it's useless to use `/R` in this line. I've replaced `\R` with `\n` and pushed the change.